org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/nexus/SolsticeScanMonitor.java

```diff
@@ -164,21 +196,48 @@ public NXcollection createNexusObject(NexusScanInfo info) {
 		
 		return scanPointsCollection;
 	}
-
+	
+	private String durationInMillisToString(Duration duration) {
+		long days = duration.toDays();
+		duration = duration.minusDays(days);
+		
+		LocalTime durationAsTime = LocalTime.MIDNIGHT.plus(duration);
+		String result = formatter.format(durationAsTime);
+		
+		if (days > 0) result = String.format("%dd ", days) + result;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)